EiDiA_backend/src/controllers/dashboard.js

```diff
@@ -1,17 +1,76 @@
 "use strict";
-
+const WidgetModel = require('../models/widget');
+const mongoose = require('mongoose');
+const LogModel = require('../models/log')
 
 const getDashboard = (req, res) => {
     res.status(200).json({response: "dummy response"});
 };
 
 const listWidgetTypes = (req, res) => {
```

Comment:
das ist ja eher dann 
```suggestion
const listWidgetsToRecord = (req, res) => {
```